Jenkinsfile

```diff
@@ -124,161 +125,175 @@ def sendNotifications(String buildStatus = '', String stageName = '', String git
 
 // begin pipeline
 try {
-	stage('Prepare') {
-		currentStage = "${env.STAGE_NAME}"
-		node {
-			// Grab the commit that triggered the build.
-			checkout scm
-
-			// Load the child Jenkinsfiles.
-			scriptForApps = load('apps/build.groovy')
-			scriptForDeploys = load('ops/deploy-ccs.groovy')
+	podTemplate(containers: [containerTemplate(name: 'bfd-cbc-build', image: 'public.ecr.aws/c2o1d8s9/bfd-cbc-build:jdk8-an29-tf12', command: 'cat', ttyEnabled: true, alwaysPullImage: true, resourceLimitCpu: '4000m', resourceLimitMemory: '8192Mi', resourceRequestCpu: '4000m', resourceRequestMemory: '8192Mi')], serviceAccount: 'bfd') {
```

Comment:
Here's where we define the pod template that Jenkins will use to create our build container environment.  I'm using approximately the same amount of resources that are available on the existing EC2 Jenkins server.  I haven't noticed any significant delays in spinning up a container of this size (I was told you were allowed to request the entire resources of a node by ITOPS).

I've already mentioned the ECR vs Artifactory discussion in the PR but noting it here.